lib/rules.js

```diff
@@ -353,6 +353,267 @@ export const GROUP_TERMS_BY_ROOT = defineRule(
 // e.g nthRoot(9, 2) -> 3
 export const NTH_ROOT_VALUE = defineRuleString('nthRoot(#a, #b)', '#eval(nthRoot(#a, #b))', {a: query.isNumber, b: query.isNumber})
 
+// FACTOR
+
+// TODO: add minus case
+// assume monic polynomial
+// TODO: add min and max to evaluate
+
+export const FACTOR_SYMBOL = defineRuleString('#a^#b_0 + ...', '#a^#eval(min(#b_0, ...)) (#a^#eval(#b_0 - min(#b_0, ...)) + ...)', {b: query.isNumber})
```

Comment:
I think this should work with subtraction as is.  Have you tried writing a test case with subtraction?